<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>property_PolylineMeasure</class>
 <widget class="QWidget" name="property_PolylineMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Polyline measurement properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>323</width>
        <height>691</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QDoubleEdit" name="VerticalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleEdit" name="HorizontalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleEdit" name="TotalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="20" column="0" colspan="3">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="15" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="segmentsLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Segments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="SegmentsUnitWithBracketLabel">
           <property name="text">
            <string>(m)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="QTableWidget" name="PointsCoordinatesTableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>75</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Area</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AreaLabelSquaredBracket">
           <property name="text">
            <string>(mÂ²)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Total Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="coordinatesLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Points coordinates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CoordinatesUnitWithBracketLabel">
           <property name="text">
            <string>(m)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="totalLabel">
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="verticalLabel">
         <property name="text">
          <string>Vertical</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleEdit" name="TotalHorizontalInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="horizontalLabel">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="3">
        <widget class="QTableWidget" name="SegmentsTableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>75</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="3">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QTableWidget" name="areaTableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>XY Area</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>YZ Area</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>XZ Area</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="18" column="0" colspan="2">
        <widget class="SubPropertyClipping" name="subPropertyClipping" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>SubPropertyClipping</class>
   <extends>QWidget</extends>
   <header>SubPropertyClipping.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>TotalInfield</tabstop>
  <tabstop>TotalHorizontalInfield</tabstop>
  <tabstop>HorizontalInfield</tabstop>
  <tabstop>VerticalInfield</tabstop>
  <tabstop>areaTableWidget</tabstop>
  <tabstop>PointsCoordinatesTableWidget</tabstop>
  <tabstop>SegmentsTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
